packages/bs-neo3/src/services/blockchain-data/DoraBDSNeo3.ts

```diff
@@ -128,6 +151,109 @@ export class DoraBDSNeo3 extends RpcBDSNeo3 {
     }
   }
 
+  async getFullTransactionsByAddress(
+    params: FullTransactionsByAddressParams
+  ): Promise<FullTransactionsByAddressResponse> {
+    this.#validateFullTransactionsByAddressParams(params)
+
+    const data: FullTransactionsItem[] = []
+
+    const { nextCursor = '', data: items } = await NeoRest.getFullTransactionsByAddress({
+      address: params.address,
+      timestampFrom: params.dateFrom,
+      timestampTo: params.dateTo,
+      network: this._network.id as 'mainnet' | 'testnet',
+      cursor: params.nextCursor,
+    })
+
+    const itemPromises = (items ?? []).map(async item => {
+      const newItem: FullTransactionsItem = {
+        txId: item.transactionID,
+        block: item.block,
+        date: item.date,
+        invocationCount: item.invocationCount,
+        notificationCount: item.notificationCount,
+        networkFeeAmount: item.networkFeeAmount
+          ? u.BigInteger.fromDecimal(item.networkFeeAmount, this._feeToken.decimals).toDecimal(this._feeToken.decimals)
+          : '0',
+        systemFeeAmount: item.systemFeeAmount
+          ? u.BigInteger.fromDecimal(item.systemFeeAmount, this._feeToken.decimals).toDecimal(this._feeToken.decimals)
+          : '0',
+        events: [],
+      }
+
+      const eventPromises = item.events.map(async event => {
+        let nftEvent: FullTransactionNftEvent
+        let assetEvent: FullTransactionAssetEvent
+        const { methodName, contractHash: hash } = event
+        const from = event.from ?? ''
+        const to = event.to ?? ''
+        const tokenId = event.tokenID ?? ''
+        const [standard] = event.supportedStandards ?? []
+        const isNep11 = standard === 'NEP-11'
+        const isNep17 = standard === 'NEP-17'
+        const isNft = isNep11 && !!tokenId && !!to && !!from
+        const [token]: TryCatchResult<Token> = await tryCatch<Token>(() => this.getTokenInfo(hash))
+        const amount = event.amount
+          ? u.BigInteger.fromNumber(event.amount).toDecimal(token?.decimals ?? event.tokenDecimals)
+          : '0'
+
+        if (isNft) {
+          const [nft]: TryCatchResult<NftResponse> = await tryCatch<NftResponse>(() =>
+            this._nftDataService.getNft({ contractHash: hash, tokenId })
+          )
+
+          nftEvent = {
+            eventType: 'nft',
+            amount,
+            methodName,
+            from,
+            to,
+            hash,
+            tokenId,
+            tokenType: isNep11 ? 'nep11' : 'generic',
```

Comment:
It is always `nep11`

Reply:
Isn't there another contract for NFT on Neo 3?